graphstorm-processing/graphstorm_processing/graph_loaders/dist_heterogeneous_loader.py

```diff
@@ -1085,8 +1085,9 @@ def write_processed_feature(
                 }
                 node_type_feature_metadata[feat_name] = node_feature_metadata_dict
 
-                feat_val = single_feature_df.take(1)[0].asDict().get(feat_name, None)
-                nfeat_size = 1 if isinstance(feat_val, (int, float)) else len(feat_val)
```

Comment:
Better to have a comment here to briefly explain about it.

Reply:
I'll add a message to the assertion that can act as an explanation as well.